react/Modal/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import styles from './index.styl'
+
+class Modal extends Component {
+  render () {
+    const {
+      title, description,
+      cancelType, cancelText, cancelAction,
+      validateType, validateText, validateAction
+    } = this.props
```

Comment:
FYI, if "secondary" and "primary" refer to button classes, they won't exist for long. There are replacements, of course, but the class name changed during the refactoring that will be released soon. That's fine like this, just wanted to warn you 😄